nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py

```diff
@@ -119,14 +120,16 @@ def init_model(self, cfg: DictConfig, trainer: Trainer):
             )
             frozen_model_cfg.activations_checkpoint_method = self.cfg.get("activations_checkpoint_method", None)
 
-        if self.trainer.precision == 'bf16':
+        if self.trainer.precision in ['bf16', 'bf16-mixed']:
+            # set hidden size in the model parallel config for pipeline parallel schedules
+            setattr(self.config, 'hidden_size', frozen_model_cfg.hidden_size)
```

Comment:
@ericharper why is this only in 'bf16' or 'bf16-mixed'? shouldn't it be a common attribute?

Reply:
good catch, I think this may have by accidentally from the rebase